modMcf/src/org/aion/mcf/config/CfgSync.java

```
@@ -72,6 +79,36 @@ public void fromXML(final XMLStreamReader sr) throws XMLStreamException {
         }
     }
 
+    private static void parseSelectedStats(List<StatsType> showStatistics, String readValue) {
+        showStatistics.clear();
+
+        String[] selected = readValue.split(",");
+
+        for (String option : selected) {
+            try {
+                showStatistics.add(StatsType.valueOf(option.toUpperCase()));
+            } catch (Exception e) {
+                // skip option
+                continue;
```

You could just have an empty catch block here